src/pages/createPledge.js

```diff
@@ -147,6 +151,13 @@ class CreatePledgePage extends React.Component {
 
   static propTypes = {
     intl: PropTypes.object.isRequired, // from withIntl
+    data: PropTypes.object,
+    name: PropTypes.string,
+    slug: PropTypes.string,
+    githubHandle: PropTypes.object,
```

Comment:
`string`?

Reply:
Yes, why?

Reply:
Seems, it's read from the query parameters in the URL and it's a string.

Reply:
Changed to string.